# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/html/HTMLCanvasElement.h
# Commit: db2347ee8147
# Full Hash: db2347ee81479323d8b188df53a9cb3c0ad6c6e7
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2021-12-10 09:10:34
# Regressor Bug: 1736177
# File Overlap Count: 2
# Description:
#   Bug 1736177 - Part 1. Rename CanvasRenderingContextHelper::GetContext to GetOrCreateContext. r=jgilbert
#   
#   This patch contains no functional changes but sets us up for refactoring
#   in a later part in this series. It also helps silence a bunch of
#   compiler warnings.
# ==============================================================================

diff -r 3dde5ddb65e5 -r db2347ee8147 dom/html/HTMLCanvasElement.h
--- a/dom/html/HTMLCanvasElement.h	Thu Dec 09 19:25:23 2021 +0000
+++ b/dom/html/HTMLCanvasElement.h	Thu Dec 09 19:25:24 2021 +0000
@@ -162,9 +162,9 @@
     SetUnsignedIntAttr(nsGkAtoms::width, aWidth, DEFAULT_CANVAS_WIDTH, aRv);
   }
 
-  virtual already_AddRefed<nsISupports> GetContext(
+  already_AddRefed<nsISupports> GetContext(
       JSContext* aCx, const nsAString& aContextId,
-      JS::Handle<JS::Value> aContextOptions, ErrorResult& aRv) override;
+      JS::Handle<JS::Value> aContextOptions, ErrorResult& aRv);
 
   void ToDataURL(JSContext* aCx, const nsAString& aType,
                  JS::Handle<JS::Value> aParams, nsAString& aDataURL,
